packages/custom_lint_core/test/configs_test.dart

```diff
@@ -144,15 +147,32 @@ custom_lint:
   rules:
   - a
 ''');
-      final configs = CustomLintConfigs.parse(analysisOptions);
+      final configs = await CustomLintConfigs.parse(analysisOptions);
 
       expect(configs.enableAllLintRules, false);
       expect(configs.rules, {
         'a': const LintOptions.empty(enabled: true),
       });
     });
 
-    test('if custom_lint.rules is present, merges with "include"', () {
+    test('include config using package: uri', () async {
+      final file = createAnalysisOptions('''
+include: $testConfigUri
+      ''');
+      final configs = await CustomLintConfigs.parse(file);
+
+      expect(configs.rules.containsKey('from_package'), true);
+    });
+
+    test('if package: uri is not resolved default to empty', () async {
+      final file = createAnalysisOptions('''
+include: $testConfigUri/notexists
+      ''');
+      final configs = await CustomLintConfigs.parse(file);
+      expect(configs, same(CustomLintConfigs.empty));
+    });
+
+    test('if custom_lint.rules is present, merges with "include"', () async {
```

Comment:
@solid-vovabeloded do you have any ideas for additional tests we could add?

Reply:
I think we have enough tests now.